crux-core/src/crux/api.clj

```diff
@@ -212,7 +215,10 @@
     (map qs/<-QueryState (.activeQueries this)))
 
   (recent-queries [this]
-    (map qs/<-QueryState (.recentQueries this))))
+    (map qs/<-QueryState (.recentQueries this)))
+
+  (slowest-queries [this]
+    (map qs/<-QueryState (.slowestQueries this))))
```

Comment:
Question - do want to grow a list fns on the main API here, or have a single stats fn that returns a map of these?

Reply:
Hey Jon - happy to go with this for now as we don't have any further additions in the pipeline - we've talked about splitting `ICruxAPI` in the past, maybe it's time to revisit that discussion?